### PR TITLE
Update invite rewards banner to explain rules of program

### LINKED ISSUE #2261: Notify users when they have invited 5 people to join Valora
### Description

Unlike v1, the next iteration of invite rewards imposes a 5-invitation limit on inviters to limit the negative impact of poor quality inviters. We have opted to forego terms and conditions for this short test, but need to notify users of the 5-invitation limit somewhere. Given that we are opting to pop a modal once the user has sent five invitations to notify them that they might not be rewarded for more invitations. 

### Acceptance Criteria [OUTDATED]
* When inviter selects a contact to invite, pop a modal to inform them of the 5-invite limit if they have already invited five people. Copy and design are here: https://www.figma.com/file/DL6Huw867n1EZOVPJ2kNmp/Invite-Rewards?node-id=1255%3A24693
* Note that we don't need to know exactly how many invites produced rewards, just the number of invitations
* In the scenario above, fire a modal impression event
* When a user clicks 'send anyway', fire a click event

UPDATE: the above AC'S have changed due to technical difficulties. On tap of a user on the send list, we need to query the blockchain to determine if the contact is already a valora user. This delay makes the modal/screen feel very laggy. As a result the implementation has been updated to the below:

### Acceptance Criteria
* Update the text on the send rewards banner to display relevant information https://valora-app.slack.com/archives/C038A310CEL/p1651869034983589

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR updates copy for the invite rewards banner.

### Other changes

Update the translation merge so that in dev mode, local translations override the OTA translations and we can actually see the changes we're making 🙈

### Tested

Manually

### How others should test

The send rewards banner should have the english text as in the screenshot.

Screenshot 2022-05-09 at 11 44 28

### Related issues

- Fixes #2261 

### Backwards compatibility
Yes

### CODE REVIEW

File: packages/mobile/src/i18n/index.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ function getAvailableResources(cachedTranslations: Resource) {
     Object.defineProperty(resources, language, {
       get: () => {
         if (!translation) {
-          translation = _.merge(value!.strings.translation, cachedTranslations[language])
+          translation = __DEV__
+            ? _.merge(cachedTranslations[language], value!.strings.translation)
+            : _.merge(value!.strings.translation, cachedTranslations[language])
```
Comment (jeanregisser) [inline packages/mobile/src/i18n/index.ts:23]: Maybe we should add a small human friendly comment? 🤖 🧑‍🚀

**Review by jeanregisser — APPROVED** [review]

File: packages/mobile/src/i18n/index.ts (hunk 1 above)
Comment (kathaypacific) [reply packages/mobile/src/i18n/index.ts]: teehee good idea, i've added :)

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest iOS Test flight Alfa jores Build, Android Alfa jores Build V1.32.0 & updated status in test rails.

- “Confirmation error” popup appears when user tries to connect phone number after creating account, so user is not able to confirm the phone number. 

**New Bug :** [link](https://github.com/valora-inc/wallet/issues/2467)

**Test rail links:**
**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/33520
**Android 10:** https://valoraapp.testrail.io/index.php?/tests/view/33535

**Devices:** iPhone 13 mini(15.1.1), Google Pixel XL (10.0)
**cc:** @kathaypacific 
Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest iOS Test flight Alfa jores Build V1.32.0 & updated status in test rails.

**Observation**

- Confirmation error is seen when user is connected to US VPN and tries to connect number
- Confirmation error is seen when user is not connected to VPN but tries to connect using US number

**New Bug :** [link](https://github.com/valora-inc/wallet/issues/2467)

**Test rail links:**
**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/33518
**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/33503

**Devices:** iPhone 13 mini(15.1.1), iPhone 12 (14.7.1)

**CC:** @kathaypacific

Thanks..!!

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android internal Alfajores Build V1.32.1 & updated status in test rails.


Test rail links:
Android 10: https://valoraapp.testrail.io/index.php?/tests/view/33535
Android 11: https://valoraapp.testrail.io/index.php?/tests/view/33550

Devices: Google Pixel 2XL(11.0), Samsung Galaxy M31(10.0)

CC: @kathaypacific

Thanks..!!

**Pull request merged**